mentoring/mentoring-events.md

```diff
@@ -0,0 +1,16 @@
+# Mentoring Activities at Events
+
+During events like KubeCon and other related events that arise, there will be the opportunity for mentoring activities. This doc will list those and their respective details. The KubeCon events are run by CNCF/LF and not Contributor Experience although we may advise and members could help out. 
+
+---
+
+CNCF is hosting another speed networking and mentoring session during [KubeCon + CloudNativeCon Europe 2018](https://events.linuxfoundation.org/kubecon-eu-2018/). 
```

Comment:
The markdown formatting on this is messed up, as it's combining into a single line.

Reply:
@cblecker hm, not for me =/

Reply:
@idvoretskyi Take a look at the github render: https://github.com/parispittman/community/blob/c648f3590439a6c5b8462ca77b55d3f3a852f094/mentoring/mentoring-events.md